src/org/sparkboard/server/slack/core.clj

```diff
@@ -59,10 +60,17 @@
      (log/debug "[web-api] POST rsp:" rsp)
      rsp)))
 
-;; TODO "when a new member joins a project, add them to the linked channel"
-
 (comment
   (http-verb "/users.list")
-  
-  ;; TODO delete/archive channel, for testing and clean-up
+
   )
+
+(def channel-name
+  (memoize
+   (fn [channel-id]
+     (get (into {}
+                (map (juxt :id :name_normalized)
+                     (:channels (web-api "channels.list"
+                                         {:auth/token (-> env/config :slack
```

Comment:
Hard-coded bot token (will fail on my machine & on staging/prod)

Reply:
should be fixed in da1f16a